### PR TITLE
oauth2 and ingress changes

### PR DESCRIPTION
**Pull request by njuettner**

### CODE REVIEW

File: helm/karma-app/templates/ingress-oauth2.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-apiVersion: extensions/v1beta1
+{{- if .Values.oauth2.enabled }}
+apiVersion: networking.k8s.io/v1
```
Comment (njuettner) [inline helm/karma-app/templates/ingress-oauth2.yaml:2]: extensions is deprecated moving to networking

File: helm/karma-app/templates/ingress.yaml [hunk 2]
```diff
@@ -6,7 +6,6 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
     kubernetes.io/tls-acme: "true"
-    nginx.ingress.kubernetes.io/auth-secret: {{ include "resource.default.name" . }}
```
Comment (njuettner) [inline helm/karma-app/templates/ingress.yaml:9]: leftover, this is not needed

**Review by calvix — APPROVED** [review]

**Review by anvddriesch — APPROVED** [review]
This looks very good! :)
By the way, do we still need to add new installations to karma? If not, maybe we should also document it when we do.

**Comment by njuettner** [comment]
> By the way, do we still need to add new installations to karma? If not, maybe we should also document it when we do.

Good point. If we have new installations, we need to add them. I'll add some docs around that.

**Pull request merged**